### PR TITLE
RF | TestUtils> overloading elementIsNotMoving(final WebElement webElement)

### PR DESCRIPTION
**Pull request by patriotby07**
https://trello.com/c/kpkvB9yI/881-rf-testutils-overloading-elementisnotmovingfinal-webelement-webelement

### CODE REVIEW

**Comment by patriotby07** [comment]
[RF | TestUtils> overloading elementIsNotMoving(final WebElement webElement)](https://trello.com/c/kpkvB9yI/881-rf-testutils-overloading-elementisnotmovingfinal-webelement-webelement)

**Comment by patriotby07** [comment]
[ERR | FolderMoveTest> testMoveFolderToFolder()](https://trello.com/c/C2ZhMcYJ/882-err-foldermovetest-testmovefoldertofolder)

File: src/test/java/runner/TestUtils.java [hunk 1]
```diff
@@ -50,5 +50,31 @@ public WebElement apply(WebDriver driver) {
...
+                private Point location = null;
+
+                @Override
+                public WebElement apply(WebDriver driver) {
+                    WebElement element;
+                    try {
+                        element = webElement;
+                    } catch (NoSuchElementException e) {
+                        return null;
+                    }
```
Comment (SergeiDemyanenko) [inline src/test/java/runner/TestUtils.java:65]: Удали эти строки

Comment (SergeiDemyanenko) [inline src/test/java/runner/TestUtils.java:65]: Удали эти строки

File: src/test/java/runner/TestUtils.java [hunk 2]
```diff
@@ -50,5 +50,31 @@ public WebElement apply(WebDriver driver) {
                 }
             };
         }
+
+        public static ExpectedCondition<WebElement> elementIsNotMoving(final WebElement webElement) {
```
Comment (SergeiDemyanenko) [inline src/test/java/runner/TestUtils.java:54]: Переименуй webElement в element

**Review by SergeiDemyanenko — APPROVED** [review]

**Review by kartavenkopavel — APPROVED** [review]

**Review by gdikhsanov — APPROVED** [review]

**Pull request merged**